sdks/java/io/jdbc/pom.xml

```diff
@@ -89,6 +89,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
+      <scope>test</scope>
```

Comment:
Yes, but shouldn't that register as an `unused declared dependency` and fail the build?